### DISCUSSION TITLE
LYWSD02MMC - future compatibility

### DISCUSSION THREAD

**Discussion by pirracas77** [pvvx/ATC_MiThermometer#420] [Q&A] [answered]
Do you think it will be any chance for the LYWSD02MMC in the future?

https://es.aliexpress.com/item/4000820010357.html

**Comment by pvvx** [comment, marked as answer]
LYWSD02MMC - https://pvvx.github.io/LYWSD02/
SoC: DA14585

Contact those who work with chips from "renesas"

https://www.renesas.com/us/en/products/wireless-connectivity/bluetooth-low-energy/da14585-smartbond-bluetooth-50-soc-audio-interface
